vyked/host.py

```diff
@@ -96,6 +99,25 @@ def _create_http_server(cls):
             return asyncio.get_event_loop().run_until_complete(task)
 
     @classmethod
+    def _create_ws_server(cls):
```

Comment:
Please remove duplication. `_create_http_server` and `_create_ws_server` have duplicate code.